TaskfMRIAnalysis/TaskfMRIAnalysis.sh

```diff
@@ -39,56 +39,92 @@
+opts_AddOptional '--lvl1fsfs' 'LevelOnefsfNames' 'DesignName_TaskName_RL@DesignName_TaskName_LR' "List of prefixes in the design.fsf filenames for each time series image for the TaskName task. Should contain same number of design files as time series images in --lvl1tasks option. (N-th design will be used for N-th time series image.) Separate multiple design files by '@' character. If no value is passed to --lvl1fsfs, the value will be set to the same list passed to --lvl1tasks."
+opts_AddOptional '--lvl2task' 'LevelTwofMRIName' 'tfMRI_TaskName' "Name of Level2 subdirectory in which all Level2 feat directories are written for TaskName. Default is 'NONE', which means that no Level2 analysis will run." 'NONE'
+opts_AddOptional '--lvl2fsf' 'LevelTwofsfName' 'DesignName_TaskName' "Prefix of design.fsf filename for the Level2 analysis for TaskName. If no value is passed to --lvl2fsf, the value will be set to the same list passed to --lvl2task."
+opts_AddOptional '--lowresmesh' 'LowResMesh' 'integer' "Default=32, which is appropriate for HCP minimal preprocessing pipeline outputs" '32'
+opts_AddOptional '--grayordinatesres' 'GrayordinatesResolution' 'integer' "Default=2mm, which is appropriate for HCP minimal preprocessing pipeline outputs" '2'
+opts_AddOptional '--origsmoothingFWHM' 'OriginalSmoothingFWHM' 'integer' "Default=2mm, which is appropriate for HCP minimal preprocessing pipeline outputs" '2'
+opts_AddOptional '--finalsmoothingFWHM' 'FinalSmoothingFWHM' 'integer' "Default=2mm, which is no additional smoothing above HCP minimal preprocessing pipelines outputs. Total desired smoothing (finalsmoothingFWHM) is reached by applying additional smoothing onto origsmoothingFWHM." '2'
+opts_AddOptional '--confound' 'Confound' 'filename' "Confound matrix text filename (e.g., output of fsl_motion_outliers). Assumes file is located in SubjectID/MNINonLinear/Results/tfMRI_TaskName_Scan. Default='NONE'" 'NONE'
+opts_AddOptional '--highpassfilter' 'TemporalFilter' 'integer' "Value for nonlinear highpass filter (in seconds). Default=200. Use 2000 for linear detrend." '200'
+opts_AddOptional '--vba' 'VolumeBasedProcessing' 'YES/NO' "Default=NO. CAUTION: Only use YES if you want unconstrained volumetric blurring of your data, otherwise set to NO for faster, less biased, and more senstive processing (grayordinates results do not use unconstrained volumetric blurring and are always produced)" 'NO'
+opts_AddOptional '--regname' 'RegName' 'RegName' "Name of surface registration technique. Default=NONE, which will use the default surface registration." 'NONE'
```

Comment:
"default surface registration" is somewhat ambiguous.  I'd say, "which will use the default (MSMSulc) surface registration".